hs/Css/Graph.hs

```diff
@@ -330,6 +330,12 @@ sidebarCSS = do
         marginBottom (px 20)
         ":hover" & do
             backgroundColor grey2
+    ".flip" & do
+        "-webkit-transform" -: "scaleX(-1)"
```

Comment:
Clay actually can generate vendor prefixes in many cases, including [transform](https://hackage.haskell.org/package/clay-0.10.1/docs/Clay-Transform.html). Try something like `transform $ scaleX (-1)`